sdk/src/androidTest/java/io/customer/sdk/util/SimpleTimerTest.kt

```diff
@@ -0,0 +1,97 @@
+package io.customer.sdk.util
+
+import androidx.test.annotation.UiThreadTest
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import io.customer.common_test.BaseTest
+import org.amshove.kluent.shouldBeFalse
+import org.amshove.kluent.shouldBeTrue
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+/**
+ * Tests must be executed on Android device, only. Running on robolectric gave false positive results.
```

Comment:
At this time, we do not have SDK tests setup to run on a real Android device. So, this test will not be executed by the CI server. 

We have RH setup today with this so we can easily setup the SDK to run tests on a real Android device.